src/components/map-page-components/map-view/MapView.tsx

```diff
@@ -1,7 +1,203 @@
 import React from 'react';
+import { Input } from 'antd';
+import { Loader } from '@googlemaps/js-api-loader';
+import styled from 'styled-components';
+
+const { Search } = Input;
+
+const StyledSearch = styled(Search)`
+  width: 40vw;
+  position: absolute;
+  z-index: 2;
+`;
+
+let map: google.maps.Map;
+
+const BOSTON: google.maps.LatLngLiteral = { lat: 42.315, lng: -71.0589 };
+const BOSTON_BOUNDS = {
+  north: 42.42,
+  south: 42.2,
+  west: -71.28,
+  east: -70.83,
+};
+
+const loader = new Loader({
+  apiKey: process.env.REACT_APP_GOOGLE_MAPS_KEY!,
```

Comment:
It tells TypeScript it will never be null or undefined, which it will never be. Otherwise it complains and don't know how to fix it otherwise
https://stackoverflow.com/questions/47704652/what-does-exclamation-point-after-variable-mean-in-javascript